protocol/proto/openyolo.proto

```diff
@@ -23,42 +23,45 @@ option java_outer_classname = "Protobufs";
 
 message CredentialRetrieveRequest {
     // at least one authMethod required
-    repeated string authMethods = 1;
-    map<string, TokenRequestInfo> supportedTokenProviders = 2;
-    map<string, bytes> additionalProps = 3;
+    repeated AuthenticationMethod authMethods = 1;
+
+    map<string, TokenRequestInfo> supportedTokenProviders = 3;
+    map<string, bytes> additionalProps = 4;
 }
 
-message CredentialRetrieveResponse {
+message CredentialRetrieveBbqResponse {
     bytes retrieveIntent = 1;
     map<string, bytes> additionalProps = 2;
 }
 
-message CredentialRetrieveResponseList {
-    repeated CredentialRetrieveResponse responses = 1;
+message CredentialRetrieveBbqResponseList {
```

Comment:
Removed. I think we used to have this because we were serializing the response list, but no longer needed.